src/cmd/services/m3dbnode/config/config.go

```diff
@@ -261,6 +261,10 @@ type BlockRetrievePolicy struct {
 	// FetchConcurrency is the concurrency to fetch blocks from disk. For
 	// spinning disks it is highly recommended to set this value to 1.
 	FetchConcurrency int `yaml:"fetchConcurrency" validate:"min=0"`
+
+	// CacheOnRetrieve enables/disables callbacks used to cache blocks fetched
+	// from disk.
+	CacheOnRetrieve *bool `yaml:"disableCacheOnRetrieve"`
```

Comment:
`yaml` should change to `cacheOnRetrieve`

Reply:
Or maybe we actually want this to be `NoCacheOnRetrieve` and then we can make it not a `*bool` so it defaults to false?

Reply:
Ah yeah, this was a bad refactor. Missed updating the yaml key. Good catch